manim/plugins/__init__.py

```diff
@@ -1,3 +1,16 @@
 from __future__ import annotations
 
-from .import_plugins import *
+from .. import config, logger
```

Comment:
```suggestion
from manim._config import config, logger
```
I believe MrDiver and Chopan were trying to move away from these kinds of imports.

Reply:
True, going with relative imports is a good practice if it's at the same level.